src/polyswarmd/__init__.py

```diff
@@ -6,14 +6,10 @@
 
 from flask import Flask, g, request
 
-from polyswarmd.config import init_config, init_logging
-log_format = os.environ['LOG_FORMAT'] if 'LOG_FORMAT' in os.environ else None
-init_logging(log_format)
+from polyswarmd.config import init_config, init_logging, require_api_key, whereami
+init_logging(os.environ.get('LOG_FORMAT'))
```

Comment:
This will throw an exception if `LOG_FORMAT` is not specified in the environment. This exception isn't handled.

Reply:
JK you used .`get` I take it back